generators/add-web-assets/exc-react/templates/src/components/About.js

```diff
@@ -0,0 +1,54 @@
+/*<% if (false) { %>
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+<% } %>
+* <license header>
+*/
+
+import React from "react";
```

Comment:
I would suggest not using semi-colons to match the backend template which try to follow standard-js, wdyt?

Reply:
and single colon

Reply:
Hmm seems the eslint job is not running when we have tpl strings 
I'll check this

Reply:
yes that's correct, until now I was running it on the generated files, and fix manually in the templates but that's not very efficient